_pages/dev/components/carousel-component.md

```diff
@@ -55,3 +55,27 @@ When there are no slides to navigate, the buttons do not appear. When a button i
 
 The number of carousel items in each slide is calculated. The calculation is driven by the minimum width per item and the available space.
 The given item width (`itemWidth`) can be specified in pixels or percentages. The calculation for the number of items per slide uses this `itemWidth` and the host element `clientWidth`. This makes the carousel reusable in different layouts (such as a one-column grid or a two-column grid), and enables the carousel to be fully responsive at the same time.
+
+### Configuring Slide Actions
+
+You can configure the actions for a slide. For example, adding Add-to-Cart buttons directly within your Merchandising Carousel to make it more efficient for consumers to add certain recommended products to their cart.
+
+You can configure which CMS components render with a product with this code:
+
+```typescript
+provideConfig({
+  cmsComponents: {
+    ProductCarouselComponent: {
+      data: {
+        composition: {
+          inner: [
+            // names of your CMS components here ex. "ProductAddToCartComponent", "ConfigureProductComponent"
+          ]
+        }
+      }
+    }
+  }
+})
+```
```

Comment:
Since the purpose of the modifications to the ProductCarousel was to add the Add to Cart button in the ProductCarousel, should we provide a code sample that shows specifically how to accomplish this.  And then perhaps we can add a note that says we can add a list of any CMS components in that config.

Reply:
Sorry, why does the above example not accomplish this?